README.md

```diff
@@ -1 +1,50 @@
+2. How do you like feedback - routine like in 1:1s, or as-it-happens
+> Timely feedback is always better for me -- just grab me!
+3. How do you prefer to receive recognition? (public or private)
+> I'm totally fine with public recognition but don't expect it!
+
+### Goals and Support
+
+1. What makes 1:1s the most valuable for you?
+> 1:1s are a time for me to voice what I'm worried about.
+2. What are your goals for this year? And for the next three months?
+> In my role as an engineering leadership sponsor I want to continue to improve our back-of-house systems' reliability and simplicity and support Casper's growth in the wholesale 
+3. What do you need from your manager?
+> I need help solving problems. I don't need help tracking against goals -- I just want clear expectations so I know where I stand and I want help solving specific problems that I come to you with. I will often start a conversation with "Here's what I'm looking for...".
+4. From your team? (this one is good if the 1:1 is with someone who manages people, but is typically relevant for IC’s, too)
+> I need clear communication (both up and down) -- tell me when you're struggling. Tell me when a deadline is slipping. Tell me when you need more support. And tell me when to back off.
+5. From your peers? (outside the team)
+> I expect clear and timely feedback -- if you disagree with my approach, or an action I took, tell me as soon as possible!
+
+### Expectations of folks who report to me
+
+Please invest in your 1:1s -- it's your time. You own your 1:1, so it is my expectation is for you to come with a topic to discuss. Here are some ideas:
+1. What are you worried about?
+2. What's some work that you've done recently that you're proud of?
+3. What's bothering you?
+
+I often will come with questions for you -- in particular, if there's an organizational or engineering focus shift, expect me to come and ask you about it.
+
+Outside of 1:1s, I expect you to track progress against the OKRs you've written each quarter. We should have a doc where we're tracking those -- please leave comments in the doc letting me know where you're at with your key results. If you are having issues with one of them, leave a note, and we'll talk about it during our 1:1. If focuses/goals need to change, that's fine! Let's just keep an open dialogue. Not meeting a key result or having to change your objectives because of shifting company or engineering priorities is fine, not meeting your goals because you forgot about them isn't okay.
+
+Everyone needs work/life balance, and my expectation is that everyone takes time off. The amount you take is up to you, but I advise folks to take 3-5 days per quarter. Make the most of your time off -- I will push you to turn off your email, Slack, and completely disconnect from work. I will lead by example here, and I expect folks to hold me accountable to that as well.
+
+### What to expect of me as your manager
```

Comment:
Does it make sense to add anything about career progression as a direct report of yours? Where do we discuss this type of thing? I know we chat about this in 1:1 periodically but does every direct report do this? I think this is an important part that is often overlooked. May be good to put it down here somewhere.

Reply:
Yep, great call. It would be very good to make that explicit in here.